e2e-testing/__tests__/orders.test.ts

```diff
@@ -0,0 +1,387 @@
+import Long from 'long';
+import {
+  BECH32_PREFIX,
+  IndexerClient,
+  IPlaceOrder,
+  LocalWallet,
+  Network,
+  Order_Side,
+  Order_TimeInForce,
+  OrderFlags, SocketClient,
+  SubaccountInfo,
+  ValidatorClient,
+  HeightResponse,
+} from '@dydxprotocol/v4-client-js';
+import {
+  DYDX_LOCAL_ADDRESS,
+  DYDX_LOCAL_ADDRESS_2,
+  DYDX_LOCAL_MNEMONIC,
+  DYDX_LOCAL_MNEMONIC_2,
+} from './helpers/constants';
+import { DateTime } from 'luxon';
+import * as utils from './helpers/utils';
+import {
+  FillTable,
+  FillType,
+  Liquidity,
+  OrderSide,
+  OrderTable,
+  SubaccountTable,
+  helpers,
+  IsoString,
+  CandleResolution,
+} from '@dydxprotocol-indexer/postgres';
+
+const PERPETUAL_PAIR_BTC_USD: number = 0;
+const quantums: Long = new Long(1_000_000_000);
+const subticks: Long = new Long(1_000_000_000);
+
+const defaultOrder: IPlaceOrder = {
+  clientId: 0,
+  orderFlags: OrderFlags.SHORT_TERM,
+  clobPairId: PERPETUAL_PAIR_BTC_USD,
+  side: Order_Side.SIDE_BUY,
+  quantums,
+  subticks,
+  timeInForce: Order_TimeInForce.TIME_IN_FORCE_UNSPECIFIED,
+  reduceOnly: false,
+  clientMetadata: 0,
+};
+
+type OrderDetails = {
+  mnemonic: string;
+  timeInForce: number;
+  orderFlags: number;
+  side: number;
+  clobPairId: number;
+  quantums: number;
+  subticks: number;
+};
```

Comment:
nit: I think some of these can be in `constants.ts`

Reply:
moved